src/groups/bmq/bmqeval/bmqeval_simpleevaluator.cpp

```diff
@@ -364,5 +364,34 @@ bdld::Datum SimpleEvaluator::Not::evaluate(EvaluationContext& context) const
     return bdld::Datum::createBoolean(!value.theBoolean());
 }
 
+// --------------------------
+// class SimpleEvaluator::Exists
+// --------------------------
```

Comment:
```suggestion
// -----------------------------
// class SimpleEvaluator::Exists
// -----------------------------
```

This is not a very important fix, just to be complete